File: src/routes/BazaarRoute/BazaarRouteProduct.cs
public class BazaarRouteProduct
{
    public string product_id;
    public BazaarRouteSummaryItem[] sell_summary;
    public BazaarRouteSummaryItem[] buy_summary;
    public BazaarRouteQuickStatus quick_status;

    public BazaarRouteProduct(string _product_id, BazaarRouteSummaryItem[] _sell_summary, BazaarRouteSummaryItem[] _buy_summary, BazaarRouteQuickStatus _quick_status)
    {
        product_id = _product_id;
        sell_summary = _sell_summary;
        buy_summary = _buy_summary;
        quick_status = _quick_status;
    }
}

File: apps/discord-bot/app/src/main/java/discordbot/hypixel/routes/BazaarRoute/BazaarRouteProduct.java
package discordbot.hypixel.routes.BazaarRoute;

public class BazaarRouteProduct {
    public String product_idd;
    /**
     * The list of the top buy orders.
     */
    public BazaarRouteSummaryItem[] buy_summary;
    /**
     * The list of the top sell orders.
     */
    public BazaarRouteSummaryItem[] sell_summary;
    public BazaarRouteQuickStatus quick_status;
}